internal/fswatcher/os_darwin.go

```diff
@@ -86,25 +81,35 @@ func New(root string, filter watchFilterFn, logf logFn, tomb *tomb.Tomb) (*FSWat
 				// before "changed" to get a behavior that is more consistent with other
 				// os_other.go.
 				switch {
-				case event.Flags&fRemoved > 0:
-					eventCh <- Event{path, OpRemoved}
-				case event.Flags&fCreated > 0:
-					eventCh <- Event{path, OpCreated}
-				case event.Flags&fChanged > 0:
-					eventCh <- Event{path, OpChanged}
+				case event.Flags&fRemoved != 0:
+					w.sendEvent(Event{path, Removed})
```

Comment:
```
./os_darwin.go:85:6: undefined: w
./os_darwin.go:87:6: undefined: w
./os_darwin.go:89:6: undefined: w
```
(also missing the `sendEvent` / `send` method)

Reply:
done